src/storage/v2/storage.hpp

```diff
@@ -267,15 +267,21 @@ class Storage final {
       return storage_->indices_.label_property_index.ApproximateVertexCount(label, property, lower, upper);
     }
 
-    std::optional<storage::IndexStats> GetIndexStats(const storage::LabelId &label,
-                                                     const storage::PropertyId &property) const {
+    std::optional<storage::LabelIndexStats> GetIndexStats(const storage::LabelId &label) const {
+      return storage_->indices_.label_index.GetIndexStats(label);
+    }
+
+    std::optional<storage::LabelPropertyIndexStats> GetIndexStats(const storage::LabelId &label,
```

Comment:
GetIndexStatsForLabelProperty?

I think we need to have consistency along these LabelProperty and Label functions